hack/.golint_failures

```diff
@@ -801,6 +801,7 @@ staging/src/k8s.io/sample-apiserver/pkg/registry/wardle/flunder
 test/e2e
 test/e2e/apimachinery
 test/e2e/apps
+test/e2e/auth
```

Comment:
Can you please make golint pass for the new code here or is there anything blocking that?